docs/code-samples/tutorials-config/0.0.3.conf

```diff
@@ -1,32 +1,32 @@
-https://github.com/digital-asset/daml-finance/releases/download/ContingentClaims.Core/1.0.0.99.20230615.1/contingent-claims-core-1.0.0.99.20230615.1.dar .lib/contingent-claims-core.dar
```

Comment:
We don't need to backport this, as it is read from `main`

Reply:
I removed it